app/schemas.py

```diff
@@ -22,14 +22,46 @@
 
 class Measurement(BaseModel):
 
-    sensor: str
-    source: str
-    pm1dot0: Optional[float] = None
-    pm2dot5: Optional[float] = None
-    pm10: Optional[float] = None
-    longitude: float
-    latitude: float
-    recorded: datetime = Field(default_factory=lambda: datetime.now(timezone.utc))
+    sensor: str = Field(
+        ...,
+        title="Sensor",
+        description="Model of the sensor device",
+    )
+    source: str = Field(
+        ...,
+        title="Source",
+        description="Name used to identify the device",
+    )
+    pm1dot0: Optional[float] = Field(
+        None,
+        title="PM1.0",
+        description="Inhalable particles that are 1.0 micrometers and smaller",
+    )
+    pm2dot5: Optional[float] = Field(
+        None,
+        title="PM2.5",
+        description="Inhalable particles that are 2.5 micrometers and smaller",
```

Comment:
The actual measure is micograms per cube meter of particles of the specific size

Reply:
Where did you take that definition? I took this from https://www.epa.gov/pm-pollution/particulate-matter-pm-basics#PM

Reply:
PM2.5 defines the particles size, but the actual measurement we record from the sensor is the concentration in ug/m3. This is from the [datasheet](https://www.espruino.com/datasheets/PMS7003.pdf) page 13, Data2 field

Reply:
How about  now? maybe is redundant but easier to understand(?)